app/config/admin/config.yml

```diff
@@ -10,3 +10,6 @@ framework:
     strict_requirements: ~
   session:
     cookie_secure: 'auto'
+
+ux_icons:
+  icon_dir: '%prestashop.admin_dir%/themes/new-theme/icons'
```

Comment:
You need to add a new line at the end of the file

Reply:
yes @jo done